BuildingBlocks/src/BuildingBlocks.API/Serilog/BackboneIdMaskingOperator.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Serilog.Enrichers.Sensitive;
+
+namespace Backbone.BuildingBlocks.API.Serilog;
+
+public class BackboneIdMaskingOperator : RegexMaskingOperator
+{
+    public BackboneIdMaskingOperator(string initials, int length = -1) : base(IsStringOfConstantLength(length) ? RegexPattern(initials, length) : RegexPattern(initials)) { }
+
+    private static bool IsStringOfConstantLength(int length) => length != -1;
+    private static string RegexPattern(string initials) => $@"{initials}[a-zA-Z0-9]*";
```

Comment:
This regex is not restrictive enough. An address can only contain lowercase characters.
But I wouldn't augment the incoming regex anyway. The caller should pass the whole regex.

Reply:
Btw: I just saw that you pass the whole regex anyway.

Reply:
Like I said above, I refactored this, hope it's ok now.